src/DocLayer.actor.cpp

```diff
@@ -165,10 +165,23 @@ ACTOR Future<Void> popDisposedMessages(Reference<BufferedConnection> bc,
 	}
 }
 
+// Splitting the delay into smaller delay(). For each delay(), DelayedTask is created and added
```

Comment:
Do you mean the queue in the main runloop of `net2.actor.cpp`?

Reply:
Yep